### PR TITLE
docs(demos): Convert css to scss and fix ripple theme bug

### LINKED ISSUE #608: Fix mdc demo site bugs for color specification
When the accent color is set to green, it seems like the ripples are purple instead of green.

I can see this in the Ripple example (at the bottom). I expect it to be green.

screen shot 2017-05-09 at 1 56 17 pm

### PR DESCRIPTION
**Pull request by amsheehan**
resolves #608

### CODE REVIEW

File: demos/demos.scss [hunk 1]
```diff
@@ -14,16 +14,17 @@
   limitations under the License
 */
 
-@import "https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css";
-
+// @import "https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css";
```
Comment (traviskaufman) [inline demos/demos.scss]: How come this is commented out?

File: demos/demos.scss [hunk 2]
```diff
@@ -14,16 +14,17 @@
   limitations under the License
 */
```
Comment (traviskaufman) [inline demos/demos.scss]: I feel like you can just put one `/* stylelint-disable */` at the top level here.

File: demos/demos.scss [hunk 3]
```diff
@@ -14,16 +14,17 @@
...
 /* Base style */
 
-:root {
-  --mdc-theme-primary: #212121;
-  --mdc-theme-accent: #64DD17;
-}
+$mdc-theme-primary: #212121;
+$mdc-theme-accent: #64dd17;
 
+@import "@material/theme/mdc-theme";
```
Comment (traviskaufman) [inline demos/demos.scss]: Why not just `@import 'material-components-web/material-components-web'` here and get rid of the import within the demo files?

File: webpack.config.js [hunk 4]
```diff
@@ -200,3 +200,31 @@ module.exports = [{
...
+    },
+    devtool: 'source-map',
+    module: {
+     rules: [{
+       test: /\.scss$/,
+       use: [{loader: 'style-loader'}].concat(CSS_LOADER_CONFIG),
+     }],
+    },
+    plugins: [
+     new ExtractTextPlugin('[name].' + (IS_PROD ? 'min.' : '') + 'css'),
```
Comment (traviskaufman) [inline webpack.config.js]: probably don't need `IS_PROD` here since we explicitly check for `IS_DEV`

File: webpack.config.js [hunk 5]
```diff
@@ -200,3 +200,31 @@ module.exports = [{
...
+if (IS_DEV) {
+  module.exports.push({
+    name: 'demo-css',
+    entry: {
+     'demo-styles': path.resolve('./demos/demos.scss'),
+    },
+    output: {
+     path: OUT_PATH,
+     publicPath: PUBLIC_PATH,
+     filename: '[name].' + 'css' + '.js',
```
Comment (traviskaufman) [inline webpack.config.js]: how about `'[name].css.js'`?

File: webpack.config.js [hunk 6]
```diff
@@ -200,3 +200,31 @@ module.exports = [{
...
+     disableHostCheck: true,
+    },
+    devtool: 'source-map',
+    module: {
+     rules: [{
+       test: /\.scss$/,
+       use: [{loader: 'style-loader'}].concat(CSS_LOADER_CONFIG),
+     }],
+    },
+    plugins: [
```
Comment (traviskaufman) [inline webpack.config.js]: Do we need this at all?

**Comment by traviskaufman** [comment]
Still seeing the incorrect ripple colors

File: demos/toolbar/waterfall-flexible-toolbar-custom-style.html [hunk 7]
```diff
@@ -96,12 +96,12 @@
           }
         }, 250);
         function init() {
-          var initialR = 255;
-          var initialG = 64;
-          var initialB = 129;
-          var endR = 63;
-          var endG = 81;
-          var endB = 181;
+          var initialR = 100;
```
Comment (amsheehan) [inline demos/toolbar/waterfall-flexible-toolbar-custom-style.html]: Updates to the new primary/accent colors

**Pull request merged**